src/algokit_utils/__init__.py

```diff
@@ -1,98 +1,71 @@
-from algokit_utils._debugging import PersistSourceMapInput, persist_sourcemaps, simulate_and_persist_response
```

Comment:
This project has pre-commit configured to run the various linting tools (black, ruff, mypy) https://github.com/algorandfoundation/algokit-utils-py/blob/main/.pre-commit-config.yaml. 

You will need to run these and fix any issues to get the PR to pass the various checks, they will also help reduce some of the unnecessary noise in the changeset due to IDE formatting differences.